beacon-chain/sync/subscriber.go

```diff
@@ -91,13 +86,12 @@ func (r *Service) registerSubscribers() {
 		r.validateAttesterSlashing,
 		r.attesterSlashingSubscriber,
 	)
-	// TODO(4154): Uncomment.
-	//r.subscribeDynamic(
-	//	"/eth2/committee_index/%d_beacon_attestation",
-	//	r.currentCommitteeIndex,                     /* determineSubsLen */
-	//	noopValidator,                               /* validator */
-	//	r.committeeIndexBeaconAttestationSubscriber, /* message handler */
-	//)
+	r.subscribeDynamic(
+		"/eth2/committee_index%d_beacon_attestation",
+		r.currentCommitteeIndex,                     /* determineSubsLen */
```

Comment:
why currentCommitteeIndex is determineSubsLen?

Reply:
committee index determines how many committees there is and a node subscribes to each committee subnet